File: src/test/java/io/bloco/faker/components/BusinessTest.java
package io.bloco.faker.components;

import org.junit.Before;
import org.junit.Test;

import java.util.Date;
import java.util.GregorianCalendar;

import io.bloco.faker.Faker;

import static org.junit.Assert.assertNotNull;
import static org.junit.Assert.assertTrue;

public class BusinessTest {

    private Faker faker;

    @Before
    public void setUp() {
        faker = new Faker();
    }

    @Test
    public void creditCardNumber() {
        assertNotNull(faker.business.creditCardNumber());
    }

    @Test
    public void creditCardExpireDate() {
        Date date = faker.business.creditCardExpireDate();
        Date today = new GregorianCalendar().getTime();
        assertTrue(date.after(today));
    }

    @Test
    public void creditCardType() {
        assertNotNull(faker.business.creditCardType());
    }
}

File: lib/src/test/kotlin/io/bloco/faker/components/BusinessTest.kt
package io.bloco.faker.components

import io.bloco.faker.Faker
import junit.framework.TestCase.assertNotNull
import junit.framework.TestCase.assertTrue
import org.junit.Test
import java.time.LocalDate

class BusinessTest {
    private val faker = Faker()

    @Test
    fun creditCardNumber() {
        assertNotNull(faker.business.creditCardNumber())
    }

    @Test
    fun creditCardExpireDate() {
        val date = faker.business.creditCardExpireDate()
        assertTrue(date.isAfter(LocalDate.now()))
    }

    @Test
    fun creditCardType() {
        assertNotNull(faker.business.creditCardType())
    }
}